docs/2.8.0/docs/canton/usermanual/apis.rst

```diff
@@ -74,13 +77,12 @@ the default JVM values will be used.
     Error messages on TLS issues provided by the networking library ``netty`` are less than optimal.
     If you are struggling with setting up TLS, please enable ``DEBUG`` logging on the ``io.netty`` logger.
 
-Note that the configuration hierarchy for a `remote participant console <https://docs.daml.com/__VERSION__/canton/scaladoc/com/digitalasset/canton/participant/config/RemoteParticipantConfig.html>`__ is slightly different
-from the in-process console or participant shown above. For configuring a remote console with TLS, please see the `scaladocs for a TlsClientConfig <https://docs.daml.com/__VERSION__/canton/scaladoc/com/digitalasset/canton/config/TlsClientConfig.html>`__
-(see also :ref:`how scaladocs relates to the configuration <configuration_reference>`).
+Note that the configuration hierarchy for a :ref:`remote participant console <canton_remote_console>` is
+slightly different from the in-process console or participant shown above.
 
 If you need to create a set of testing TLS certificates, you can use the following openssl commands:
 
-.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/gen-test-certs.sh
+.. literalinclude:: /canton/includes/mirrored/community/app/src/pack/config/tls/gen-test-certs.sh
```

Comment:
Are you sure this change is correct / compatible with what we have in Canton ?

Reply:
yes! i moved our helper script into the public docs